components/Footer/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+import Link from "next/link";
+import React from "react";
+import { Logo } from "../Logo/Logo";
+
+export interface Link {
+  label: string;
+  route: string;
+}
+
+interface FooterLinkProps extends React.HTMLProps<HTMLLIElement> {
+  route: string;
+}
+
+interface FooterProps extends Partial<React.HTMLProps<HTMLElement>> {
+  links: Link[];
+}
+
+const textClasses =
+  "font-outfit text-[0.875rem] text-center font-semibold text-white";
+
+const focusClasses = "focus:shadow-md focus:border-white focus:border-2";
+
+const disabledClasses = "disabled:cursor-not-allowed disabled:bg-gray";
+
+export const Footer = ({ links }: FooterProps) => {
+  return (
+    <footer
+      className="bg-blue pb-14 pt-10 md:pt-14"
+      aria-label="Footer"
+      data-testid="footer"
+    >
+      <nav className="mx-auto flex w-full max-w-[1040px] flex-col items-center justify-between md:flex-row">
```

Comment:
Here `w-full` is not doing anything, `max-w-[1040px]` can be `max-w-5xl` instead